gateway/src/apicast/policy/3scale_referrer/3scale_referrer.lua

```diff
@@ -0,0 +1,20 @@
+local policy = require('apicast.policy')
+local _M = policy.new('3scale Referrer policy')
+
+local new = _M.new
+
+function _M.new(config)
```

Comment:
We don't even need to override new, right?

Reply:
You're right. This is the first policy we have with an empty configuration, so we don't need to override `new`. Didn't realize that :+1:

Reply:
Fixed.